packages/test-utils/src/wrapper.js

```diff
@@ -67,6 +74,17 @@ export default class Wrapper implements BaseWrapper {
     ) {
       this.isFunctionalComponent = true
     }
+
+    const { autoDestroy } = config
+    if (autoDestroy) {
+      if (autoDestroy instanceof Function) {
+        autoDestroy(destroyAllInstances)
```

Comment:
This line is wrong. It will for example result in `describe > it > shallowMount > new Wrapper > afterEach`. That way we have a lot of `afterEach` calls insides `it` while we only want a single one *outside*.